amlip_docs/rst/user_manual/nodes/inference_datatype.rst

```diff
@@ -0,0 +1,21 @@
+.. include:: ../../exports/alias.include
+
+.. _user_manual_datatype_inference:
+
+Inference Data Type
```

Comment:
This file is not referred anywhere, a new section must be added with types.
However, I am not sure we must have such section as it is a work in progress, and the information giving here is not much useful for a user as this data type actually reference everything, not a partial data-set.

Reply:
Not sure either :/